kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/swagger/SwaggerIntegrationTest.java

```diff
@@ -53,39 +57,49 @@ public static void closeHttpClient() {
 
     @Test
     public void testSwaggerDocs() throws Exception {
-        String html = invokeGet(getContextRoot()+"docs/");
+        Response response = invokeGet(getContextRoot(3)+"docs/");
```

Comment:
Can you please close the response in the end of test?

Reply:
done